content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -79,75 +72,59 @@ npm install
 ```terminal
 ls -1
 README.md
-functions
+functions/
 netlify.toml
-node_modules
+node_modules/
 package.json
-prisma
-public
-schema.sql
+prisma/
+public/
 ```
 
-## 3. Set the DATABASE_URL environment variable locally
+## 3. Get the database connection string
 
-Set the `DATABASE_URL` environment variable locally so you can create the database schema and Prisma can access the database to introspect:
+You need the database connection string to define the `DATABASE_URL` environment variable.
 
-```terminal
-export DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-```
+If you are using Heroku, follow the steps below.
 
-> **Note:** you will need the `DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
+1. In Heroku, open your app and select the **Resources** tab.
+2. Click the **Heroku Postgres** database add-on.
+3. Select the **Settings** tab in the page of the Heroku Postgres add-on.
+4. Click **View Credentials**.
+5. From the **URI** field, copy the entire connection string.
 
-You need to replace the uppercase placeholders with your database credentials, e.g.:
+**Checkpoint:** The Heroku connection string has the following format:
```

Comment:
This can also be the local connection string if they are not using Heroku.

Reply:
Removed the mention of Heroku here.